### ISSUE TITLE
[Bug] There is no option to import browser data from firefox

### ISSUE THREAD

**Issue by Cubifyy** [Floorp-Projects/Floorp#744]
### Attention!

- [X] I understand that Floorp is NOT supported on Windows 7/8.1 or lower.

### Describe the bug

Upon install, I tried to import browser data from Firefox to Floorp, but Firefox doesn't appear as an option.

### To Reproduce

1. Open Floorp for the first time OR go to Settings>General>Import Browser Data

### Expected behavior

Have firefox as an option to import browsing data from.

### Screenshots

![image](https://github.com/Floorp-Projects/Floorp/assets/122413885/d5de81c3-caf8-4fbd-abaa-170a5d4efd26)

### OS Info

Windows 11 22H2 Build 22621.3007

### Floorp Version

11.8.1

**Comment by CutterKnife** [comment]
This is probably the intended behavior (given that Floorp is based on Firefox), since data is rarely migrated from Firefox to Firefox.

**Comment by Cubifyy** [comment]
Well that’s a bit annoying, Firefox is a popular browser and there should
be an option to import from it

**Comment by TheChilledBuffalo** [comment]
@Cubifyy You could just sign into your Firefox account. That should bring in all of the stuff except Form autofill data I think.

![image](https://github.com/Floorp-Projects/Floorp/assets/87632612/20e60222-88d3-4fc3-ba9b-b8622e0ece73)

**Comment by aaronsql2019** [comment]
I did NOT know that signing into my Mozilla account would fix this.
THANKS for that information! that is EXACTLY what I wanted to hear!

**Comment by wilenius** [comment]
I have signed to my Mozilla account, and the password Sync is on. However, the passwords are not synced to Floorp.

Weirdly, I just created a new password in Floorp, and it appears in Firefox. When I delete it in Firefox, it also gets deleted in Floorp.

But the old passwords from Firefox are still not synced. Any ideas? Should I create a separate issue for this?

I'm using Floorp for Flatpak, 11.19.1 (64-bit), with Ubuntu 20.04.
